verification/verify_meter.py

```
@@ -0,0 +1,99 @@
+import time
+import subprocess
+import os
+import signal
+import sys
+from playwright.sync_api import sync_playwright
+
+def verify_live_monitor_meter():
+    # Start the mock app in the background
+    # We use setsid to create a new process group so we can kill the whole tree later
+    server_process = subprocess.Popen(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)